src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,50 @@
+import "./_progress-bar.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface ProgressBarProps {
+  percentage: number;
+  style: {
+    trackColor?: string;
+    backgroundColor?: string;
+    completedColor?: string;
+  };
+  children?: React.ReactNode;
+  customClassName?: string;
+}
+
+function ProgressBar(props: ProgressBarProps) {
+  const {
+    percentage,
+    style: {trackColor = "blue", backgroundColor = "gray", completedColor = "green"},
+    children,
+    customClassName
+  } = props;
+  const progressBarContainerClassName = classNames(
+    "progress-bar-container",
+    customClassName
+  );
+
+  return (
+    <div className={progressBarContainerClassName}>
```

Comment:
Let's remove this redundant wrapper div and any related styles attached to it on the SCSS file